### ISSUE TITLE
[Feature request] Add Crypto Notepad to "Send to" - when right click on a text files

### ISSUE THREAD

**Issue by Sigmanor** [Sigmanor/Crypto-Notepad#11]
Add Crypto Notepad to "Send to" - when right click on a text files.
![newfeature](https://user-images.githubusercontent.com/7986067/49691574-52349a00-fb4e-11e8-8294-a08b786ca196.png)

**Comment by smaragdus** [comment]
@Sigmanor 
This can be done manually.

**Comment by Sigmanor** [comment]
@smaragdus 
Yes, you can just create shortcut in `%APPDATA%\Microsoft\Windows\SendTo` folder.
But this will work only for `.cnp` files. It would be nice to open any text files in this way.
